modules/oadp-restic-issues.adoc

```diff
@@ -23,14 +23,15 @@ You can resolve this issue by creating a supplemental group for `Restic` and add
 
 . Create a supplemental group for `Restic` on the NFS data volume.
 . Set the `setgid` bit on the NFS directories so that group ownership is inherited.
-. Add the `spec.configuration.restic.supplementalGroups` parameter and the group ID to the `DataProtectionApplication` manifest, as in the following example:
+. Add the `spec.configuration.nodeAgent.supplementalGroups` parameter and the group ID to the `DataProtectionApplication` manifest, as in the following example:
 +
 [source,yaml]
```

Comment:
Unfortunately your example is not compliant with the OCP docs contributor guidelines at https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting-for-kubernetes-and-openshift-api-objects . Briefly, you need to provide a "complete" YAML with apiVersion, kind, and the full hierarchy to the keys. It is a common "tripwire"; I would recommend fixing any example that you modify.

Reply:
Done. Thanks.